teammapper-backend/src/map/entities/mmpMap.entity.ts

```diff
@@ -16,6 +16,9 @@ export class MmpMap {
   @Column({ type: 'timestamptz', nullable: true, default: () => 'now()' })
   lastModified: Date
 
+  @Column({ type: 'timestamptz', nullable: true })
```

Comment:
maybe also add the default here like in lastModified

Reply:
on second thought, maybe better not to use the default, as all existing maps will then also get a lastAccessed for the actual time? Could you verify this?

Reply:
Yeah my thought was just because a map is created it isn't immediately accessed (but it _is_ immediately modified), so I didn't add a default.